test/tools/unified-spec-runner/operations.ts

```diff
@@ -659,6 +672,42 @@ operations.set('runCommand', async ({ entities, operation }: OperationFunctionPa
   return db.command(command, options);
 });
 
+operations.set('runCursorCommand', async ({ entities, operation }: OperationFunctionParams) => {
+  const db = entities.getEntity('db', operation.object);
+  const { command, ...opts } = operation.arguments!;
+  const cursor = db.runCursorCommand(command, {
+    readPreference: ReadPreference.fromOptions(opts),
```

Comment:
this `ReadPreference` construction should follow the same pattern as in `createCommandCursor`

Reply:
fixed